### DISCUSSION TITLE
No module named 'airflow.providers'

### DISCUSSION THREAD

**Discussion by abhi8585** [apache/airflow#24750] [Q&A]
I have an Airflow instance in my local environment. I am upgrading from version 1.10.5 to 2.0.0.
I am following the steps mentioned in https://airflow.apache.org/docs/apache-airflow/stable/upgrading-from-1-10/index.html. I have upgraded my airflow instance up to 1.10.15.
I have made the changes in DAGS as suggested by airflow upgrade scripts regarding replacing the traditional imports of operators with backport providers.

I am working with "**apache-airflow-backport-providers-postgres**". I have installed it with **pip install apache-airflow-backport-providers-postgres**.

After performing the above steps when I run my airflow instance I got this error on the UI "**No module named 'airflow.providers**". PFA screenshot for more information.
![Screenshot (141)](https://user-images.githubusercontent.com/22455430/176651119-6cab5e49-70d0-48d1-b7ff-321c16e203db.jpg)

![Screenshot (141)](https://user-images.githubusercontent.com/22455430/176651020-b848e937-d998-495a-8e16-4b920e301105.jpg)

As mentioned about creating the symbolic link at https://airflow.apache.org/docs/apache-airflow/1.10.14/backport-providers.html . I have created a symbolic link to the provider's folder where my backport packages are installed.

Any help will be appreciated

**Comment by potiuk** [comment]
Backport providers were only for 1.10 (this is what the "backport" indicates) . You should NOT install it because it will effectively re-install Airflow 1.10. 

You should have "apache-airflow-providers-postgres" installed.
